apim/3.x/README.adoc

```diff
@@ -150,18 +186,37 @@ define the following configuration options:
 |`+mongo.socketKeepAlive+` |Enable keep alive for socket |`+false+`
 |===
 
-==== Mongo ReplicaSet
+==== Mongo Replica Set
 
 [cols=",,",options="header",]
 |===
 |Parameter |Description |Default
-|`+mongodb-replicaset.enabled+` |Enable deployment of Mongo replicaset
+|`+mongodb-replicaset.enabled+` |Enable deployment of Mongo replica set
 |`+false+`
 |===
 
 See
 https://github.com/helm/charts/tree/master/stable/mongodb-replicaset[MongoDB
-replicaset] for detailed documentation on helm chart.
+replica set] for detailed documentation on Helm chart.
+
+
+==== PostgreSQL Database (via JDBC Connection) 
+To install a new PostgresSQL database, use the command below and update the `username`, `password`, and `databasename` parameters: 
+----
+helm install --set postgresqlUsername=postgres --set postgresqlPassword=P@ssw0rd 
```

Comment:
Here, we provide the helm command to install postgres.
But in the mongo part, we do not provide the helm command to install mongo.
Which would be, for example `helm install mongodb bitnami/mongodb --set auth.rootPassword=r00t`